packages/testsweets/lib/src/ui/testsweets_overlay/testsweets_overlay_view.dart

```diff
@@ -28,8 +28,8 @@ class TestSweetsOverlayView extends StatelessWidget {
 
   @override
   Widget build(BuildContext context) {
-    return Material(
-      child: enabled
+    return MaterialApp(
```

Comment:
This is a very interesting change. This means we have 2 navigators, overlay handlers, and everything required by an app running. Why did we change this to a `MaterialApp`

Reply:
I did this change mainly to fix the issue that arose from moving the `TestsweetsOverlay` from the builder and making it the parent of the client app

Reply:
Got it. I think this has performance issues, maybe. But I might be wrong. If it's good then I'm happy with it!